### PR TITLE
Show RBAC errors if any

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: api/src/preflight/preflight_store.ts [hunk 1]
```diff
@@ -4,6 +4,11 @@ import { Params } from "../server/params";
 import { PreflightResult } from "./";
 import { ReplicatedError } from "../server/errors";
 
+interface PreflighParams {
```
Comment (marccampbell) [inline api/src/preflight/preflight_store.ts]: spelling?

File: migrations/tables/app_downstream_version.yaml [hunk 2]
```diff
@@ -41,5 +41,10 @@ spec:
         type: text
       - name: preflight_result_created_at
         type: timestamp without time zone
+      - name: preflight_ignore_permissions
+        type: boolean
+        default: "false"
+        constraints:
+          notNull: true
```
Comment (marccampbell) [inline migrations/tables/app_downstream_version.yaml]: fyi, this won't migrate....  we don't yet support migrations to default w/ not null in schemahero.

i don't know if we need this to migrate?

**Comment by divolgin** [comment]
This needs a new troubelshoot release before merging

File: api/Dockerfile.skaffold [hunk 3]
```diff
@@ -41,7 +41,7 @@ ADD ./src /src/src
 
 RUN make build
 
-# COPY ./kots.so /lib/kots.so
+COPY ./kots.so /lib/kots.so
```
Comment (divolgin) [inline api/Dockerfile.skaffold]: no! :)

File: web/src/components/PreflightResultPage.jsx [hunk 4]
```diff
@@ -207,6 +258,19 @@ export default compose(
       };
     }
   }),
+  graphql(getPreflightCommand, {
```
Comment (divolgin) [inline web/src/components/PreflightResultPage.jsx]: this is not defined.  i've removed it in the rebased branch

**Comment by divolgin** [comment]
new PR created for this

**Pull request closed without merging**